friends/src/App.js

```diff
@@ -23,6 +28,28 @@ class App extends Component {
         })
   }
 
+  handleChange = e => {
```

Comment:
Would add a checker for age to make sure it is number instead of a string. This could break the database if the information is not passed correctly.